integration-cli/docker_cli_cp_test.go

```diff
@@ -374,6 +374,8 @@ func TestCpUnprivilegedUser(t *testing.T) {
 }
 
 func TestCpVolumePath(t *testing.T) {
```

Comment:
@tianon agreed. but the part you're gonna gain by splitting is what other cp tests like TestCpRelativePath, TestCpAbsolutePath, TestCpToDot are already testing?